doc/programmers/documentation.rst

```diff
@@ -37,12 +37,15 @@ appropriate ``.rst`` file.
 How does this work?
 -------------------
 
-To have an offline version of the documentation just issue ``make doc`` in the
-build directory.  The HTML will be stored in ``doc/html``. Open the
-``doc/html/index.html`` file with your browser to see and browse the
+To have an offline version of the documentation just issue 
+``sphinx-build doc/ _build/.``.  The HTML will be stored in ``_build/``. 
+Open the ``_build/index.html`` file with your browser to see and browse the
 documentation.
```

Comment:
With `make doc` the documentation building process was run with `sphinx-build -b html -d doc/doctrees ${PROJECT_SOURCE_DIR}/doc doc/html` Did you make sure this one works exactly the same?

Reply:
- `-b html` is default. 
- `-d` (path for the cached environment and doctree files) also has a default. Not sure how important this is.
- `${PROJECT_SOURCE_DIR}/doc doc/html` is just sourcedir and outdir.

Thus, pretty sure the result will be the same.